src/gui/accountstate.h

```diff
@@ -221,6 +224,7 @@ protected Q_SLOTS:
      */
     AccountAppList _apps;
 
+    UserStatus *_userStatus;
```

Comment:
Any particular reason this is a pointer instead of being on the stack?

Reply:
Consistency I'd say (RemoteWipe is not on the stack either for instance).

"Interestingly" RemoteWipe is leaked btw as it got no parent.

Reply:
I am not sure how to change this...

Reply:
I'd say don't change it. ;-)